### ISSUE TITLE
One gamepad setting for all consoles!

### ISSUE THREAD

**Issue by Roper89** [Swordfish90/Lemuroid#926]
There are no separate gamepad settings for each console.
It is impossible to change controls every time you change the game from NES to Sega.

**Please make separate profiles for the gamepad!**

**Comment by master1274** [comment]
Of course, this is an important functionality for the emulator. Separate profile settings are required.
